### ISSUE TITLE
Serialization exception with Orleans 7.1

### ISSUE THREAD

**Issue by jkonecki** [OrleansContrib/OrleansDashboard#389]
Dashboard fails to show any data due to the following exception being thrown from:

OrleansDashboard.IDashboardGrainOrleansDashboard.IDashboardGrain.GetCounters()
OrleansDashboard.IDashboardGrainOrleansDashboard.IDashboardGrain.GetClusterTracing()

```
System.TypeLoadException: Could not load type 'Orleans.Serialization.Codecs.TagDelimitedFieldCodec' from assembly 'Orleans.Serialization, Version=7.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Orleans.Runtime.Messaging.MessageSerializer.Write(PipeWriter writer, Message message) in /_/src/Orleans.Core/Messaging/MessageSerializer.cs:line 211
   at Orleans.Runtime.Messaging.Connection.ProcessOutgoing() in /_/src/Orleans.Core/Networking/Connection.cs:line 409
   at Orleans.Serialization.Invocation.ResponseCompletionSource`1.GetResult(Int16 token) in /_/src/Orleans.Serialization/Invocation/ResponseCompletionSource.cs:line 230
   at System.Threading.Tasks.ValueTask`1.ValueTaskSourceAsTask.<>c.<.cctor>b__4_0(Object state)
--- End of stack trace from previous location ---
   at OrleansDashboard.DashboardClient.DashboardCounters()
   at OrleansDashboard.DashboardMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
```

Running Orleans Dashboard version v.7.0.2.0.

Are there any plans for recompiling Orleans Dashboard against Orleans 7.1? Should I prepare a PR?

**Comment by richorama** [comment]
Thanks Jakub! If you're happy to prepare a PR, I would be happy to merge!
Otherwise I'll stick it on my todo list :¬)

**Comment by jkonecki** [comment]
Submitted PR #390

**Comment by jkonecki** [comment]
I'll test the new NuGet package version as soon as you push it.

**Comment by jkonecki** [comment]
Issue is fixed in OrleansDashboard 7.1.0.
